modules/client-configuration/pages/clients-rh-cdn.adoc

```diff
@@ -161,6 +157,7 @@ The products you need for this procedure are:
 | OS Version | Product Name
 | {redhat} 7 | RHEL7 Base x86_64
 | {redhat} 8 | RHEL or SLES ES or CentOS 8 Base
+| {redhat} 9 | RHEL 9 Base
```

Comment:
I'm not fully sure if that's the correct name, I think we included Liberty word on it.